### PR TITLE
[UI] added trim option

### PR DESCRIPTION
**Pull request by KevinHock**
This is an attempt to fix https://github.com/python-security/pyt/issues/46. Let me know what you think @Thalmann.

### CODE REVIEW

File: pyt/vulnerabilities.py [hunk 1]
```diff
@@ -224,7 +236,8 @@ def get_vulnerability(source, sink, triggers, lattice):
     """
     source_in_sink = lattice.in_constraint(source.cfg_node, sink.cfg_node)
 
-    secondary_in_sink = []
+    secondary_in_sink = list()
```
Comment (Thalmann) [inline pyt/vulnerabilities.py]: +1 :)

**Comment by Thalmann** [comment]
Looks good :)
Good renaming!

I shortly glimpsed over the additions, what is the idea with the evil_node and what is its purpose?

**Comment by KevinHock** [comment]
Good catch, I couldn't really think of a good name for it. I hope this commit makes it more clear https://github.com/python-security/pyt/pull/47/commits/bd33820c564e3727494671c7e63b55913819ba2a. That location is where we step backwards and find the nodes to add to the trimmed list. I can't think of a good one-size-fits-all variable name so I did `node_in_the_vulnerability_chain = secondary_node_in_sink_args` to make it more clear, what are your thoughts on this? :/

**Comment by KevinHock** [comment]
Going to make a blackbox PR that depends on this code, to deal with https://github.com/python-security/pyt/issues/45.

**Pull request merged**